packages/systemd/systemd.spec

```diff
@@ -242,6 +268,7 @@ install -p -m 0644 %{S:4} %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/i
 %{_cross_bindir}/systemd-cat
 %{_cross_bindir}/systemd-cgls
 %{_cross_bindir}/systemd-cgtop
+%{_cross_bindir}/systemd-creds
```

Comment:
Are we planning to [pass credentials](https://man7.org/linux/man-pages//man1/systemd-creds.1.html) to systemd units? Otherwise, should this be excluded with `%exclude` or installed in other subpackage?

Reply:
I'm interested in this as a potential future direction, and wanted it enabled to facilitate that. Similar story to `systemd-repart` which we ended up using.